src/gui/src/layoutViewer.cpp

```diff
@@ -2921,6 +2921,7 @@ void LayoutViewer::addMenuAndActions()
   menu_actions_[CLEAR_HIGHLIGHTS_ACT] = clear_menu->addAction(tr("Highlights"));
   menu_actions_[CLEAR_RULERS_ACT] = clear_menu->addAction(tr("Rulers"));
   menu_actions_[CLEAR_FOCUS_ACT] = clear_menu->addAction(tr("Focus nets"));
+  menu_actions_[CLEAR_GUIDES_ACT] = clear_menu->addAction(tr("Route Guides"));
```

Comment:
I think this is okay for now, but we probably need a better way to add these for the renderers so the code doesn't get too much more co-mingled.